lib/parser.js

```diff
@@ -46,14 +50,22 @@ function parse(opts, cb) {
                                 'opts.config');
     }
 
+    if (!opts.configPath && !opts.basePath) {
+        throw new verror.VError('must specify opts.basePath');
```

Comment:
You can just use asserf.fail('msg'), this will give you the same assertion error that assert-plus throws.